operator/scripts/update-kcc-manifest/main.go

```diff
@@ -459,7 +459,7 @@ func swapContainerRegistry(manifestPath string) error {
 		return fmt.Errorf("error reading manifestPath: %w", err)
 	}
 	manifest := string(content)
-	updatedManifest := strings.ReplaceAll(manifest, "gcr.io/cnrm-eap/", "gcr.io/gke-release/cnrm/")
+	updatedManifest := strings.ReplaceAll(manifest, "gcr.io/cnrm-eap/", "gcr.io/gke-release/")
```

Comment:
Could you elaborate on the triaging process? 

This line of code is not changed since it is open sourced 2 years ago. https://github.com/GoogleCloudPlatform/k8s-config-connector/blame/master/operator/scripts/update-kcc-manifest/main.go#L462 

The duplicate `cnrm` is recently failing. So I'm not sure if this is the root cause. 
Also, the propose-tag is the new release and approved working in 1.126. I suggest we do not change its release path.